docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -1489,8 +1489,7 @@ Parameters:
     handler](#timeout-handler).
 -   `callback_timeout_secs`: set a timeout for the `timeout` block. Defaults to
     15 minutes.
--   `share_processes`: by default, processes and Docker state changes
```

Comment:
We do still clean up docker things it look like https://github.com/evergreen-ci/evergreen/blob/8db6ef31056a6da3f7b2b61b4e565be8753bced3/agent/agent.go#L1284

Reply:
Oh interesting. Should I remove this docker cleanup? It looks intended to cleanup our resources. I could see a world where a user script or service is using docker and they might be relying on this cleanup indirectly (to not clog up hosts?). What do you think?